Dockerfile

```diff
@@ -32,10 +33,11 @@ RUN curl -LO ${SBCL_URL} && \
     tar -xf sbcl-${SBCL_VERSION}-source.tar.bz2 && \
     rm /src/sbcl-${SBCL_VERSION}-source.tar.bz2 && \
     cd /src/sbcl-${SBCL_VERSION} && \
-    bash make.sh --fancy && \
+    ln -s /src/sbcl-${SBCL_VERSION} /src/sbcl && \
+    bash make.sh --fancy --with-sb-dynamic-core --with-sb-linkable-runtime && \
+    (cd src/runtime && make libsbcl.a) && \
```

Comment:
why is this being run in a subshell?

Reply:
cwd is automatically restored to what it was before the subshell. I thought that was cute. Maybe there are subtlties to subshells that I'm not aware of though.